app/antedecorators/gas.go

```diff
@@ -8,14 +8,10 @@ import (
 
 func GetGasMeterSetter(pk paramskeeper.Keeper) func(bool, sdk.Context, uint64, sdk.Tx) sdk.Context {
 	return func(simulate bool, ctx sdk.Context, gasLimit uint64, tx sdk.Tx) sdk.Context {
-		if ctx.BlockHeight() == 0 {
```

Comment:
would `CosmosGasParams` be set at this point when block height is 0? Just want to make sure it won't panic if we remove this

Reply:
Yea the GetCosmosGasParams return the default values if it’s not set